### DISCUSSION TITLE
Using SAINTexpress in prolfqua

### DISCUSSION THREAD

**Discussion by Suomwe** [fgcz/prolfqua#87] [Q&A]
Hello,

I would like to use SAINTexpress in prolfqua to analyse my data but the vignettes and tutorials do not include any details on this, just that this sort of analysis is possible. 

I know that 'runSaint' is the way to do this, would anyone be able to shed some light on this? 

thanks!

**Comment by wolski** [comment]
Dear @Suomwe 
Regarding SAINTExpress, please have a look at the https://github.com/prolfqua/prolfquasaint
package, where I moved all the code related explicitly to SAINT.
Install the package and then have a look at:

https://github.com/prolfqua/prolfquasaint/blob/main/inst/application/SE2_DIANN/DIANN_SE.R

Otherwise, if you are working with fragpipe outputs, you can also have a look at FragPipeAnalyser, which AFAIK has SaintExpress support.
